kratos/python_scripts/analysis_stage.py

```diff
@@ -60,12 +60,29 @@ def RunSolutionLoop(self):
         """This function executes the solution loop of the AnalysisStage
         It can be overridden by derived classes
         """
+        # We check of some model part requires reinitialize
+        modified_model = False
+        if hasattr(self, 'model'):
+            model_part_names = self.model.GetModelPartNames()
+            for model_part_name in model_part_names:
+                if self.model.GetModelPart(model_part_name).Is(KratosMultiphysics.MODIFIED):
+                    modified_model = True
+                    break
+
+        # Solution loop
+        solver = self._GetSolver()
         while self.KeepAdvancingSolutionLoop():
-            self.time = self._GetSolver().AdvanceInTime(self.time)
+            self.time = solver.AdvanceInTime(self.time)
+            # We reinitialize if remeshed previously
```

Comment:
the tag 'modified_model' is not changing its value during the simulation...

Reply:
True

Reply:
I modified, sorry, I run a test with just one step...